main.cpp

```diff
@@ -474,6 +537,22 @@ Thing 10) Cruise Ship
     3) shopping complex opens all day
  */
 
+struct CruiseShip
+{
+    Staff name;
+    BuildSpecification buildSpecsOfShip;
+    Entertainment typeOfEntertainment;
+    Facilities typeOfFacilities;
+    SafetyProcedure shipSafetyProcedure;
+
+    // travel for 10 days without stopovers
+    void travelFor10DaysWithoutStopovers();
+    // ferry 5000 passengers
+    void ferry5000Passengers();
+    // shopping complex opens all day
+    void shoppingComplexOpensAllDay();
```

Comment:
These function names don't match the descriptions or the description doesn't describe an action.
Function names should describe the actions they perform. 
They should have verbs in their name
- [ ] revise